Gopkg.toml

```diff
@@ -47,8 +64,8 @@ required = [
 [[constraint]]
   name = "github.com/operator-framework/operator-sdk"
   # The version rule is used for a specific release and the master branch for in between releases.
-  # branch = "master" #osdk_branch_annotation
-  version = "=v0.3.0" #osdk_version_annotation
+  branch = "v0.7.x" #osdk_branch_annotation
```

Comment:
should this be v0.7.0 instead?

Reply:
You're right, I wasn't going off the tag. Fixed and being pushed now.